cmd/meroxa/root/root.go

```diff
@@ -85,28 +85,48 @@ resource types:
 	if v, ok := os.LookupEnv("MEROXA_V2"); !ok || ok && v != "only" {
 		// TODO: Once we make a full transition to `subject-verb-object` remove the `deprecated` pkg altogether
 		deprecated.RegisterCommands(cmd)
+
+		cmd.AddCommand(builder.BuildCobraCommand(&auth.Login{}))
+		cmd.AddCommand(builder.BuildCobraCommand(&auth.Logout{}))
+		cmd.AddCommand(builder.BuildCobraCommand(&auth.WhoAmI{}))
 	}
 
 	// v2
 	cmd.AddCommand(CompletionCmd()) // Coming from Cobra
 
 	cmd.AddCommand(builder.BuildCobraCommand(&api.API{}))
-	cmd.AddCommand(builder.BuildCobraCommand(&auth.WhoAmI{}))
-	cmd.AddCommand(builder.BuildCobraCommand(&auth.Login{}))
-	cmd.AddCommand(builder.BuildCobraCommand(&auth.Logout{}))
 	cmd.AddCommand(builder.BuildCobraCommand(&billing.Billing{}))
 	cmd.AddCommand(builder.BuildCobraCommand(&connectors.Connect{}))
 	cmd.AddCommand(builder.BuildCobraCommand(&open.Open{}))
 	cmd.AddCommand(builder.BuildCobraCommand(&version.Version{}))
 
 	// New commands following `subject-verb-object` only shown if using `MEROXA_V2`)
 	if _, ok := os.LookupEnv("MEROXA_V2"); ok {
+		cmd.AddCommand(builder.BuildCobraCommand(&auth.Auth{}))
 		cmd.AddCommand(builder.BuildCobraCommand(&connectors.Connectors{}))
 		cmd.AddCommand(builder.BuildCobraCommand(&endpoints.Endpoints{}))
 		cmd.AddCommand(builder.BuildCobraCommand(&pipelines.Pipelines{}))
 		cmd.AddCommand(builder.BuildCobraCommand(&resources.Resources{}))
 		cmd.AddCommand(builder.BuildCobraCommand(&transforms.Transforms{}))
+
+		setAliases(cmd)
 	}
 
 	return cmd
 }
+
+// setAliases includes command to root but not shown in help
+// e.g.: `meroxa login` -> `meroxa auth login`.
+func setAliases(cmd *cobra.Command) {
+	aliases := map[string]builder.Command{
+		"login":  &auth.Login{},
+		"logout": &auth.Logout{},
+		"whoami": &auth.WhoAmI{},
+	}
+
+	for _, c := range aliases {
+		cc := builder.BuildCobraCommand(c)
```

Comment:
Note: I could include something like this to make more explicit. I didn't because those three auth commands happen to have the same "Use".

```
for v, c := range aliases {
		cc := builder.BuildCobraCommand(c)
                 cc.Use = v
...
}
```

Reply:
TBH I would use the code in your comment, it doesn't hurt to have it, then we can sleep just a bit easier knowing that if we add more aliases in the future we won't have an unexpected result 🤷